src/device/rdk/interface.rs

```diff
@@ -579,6 +579,19 @@ pub fn get_thunder_property(method_name: &str, key_name: &str) -> Result<String,
     }
 }
 
+pub fn get_frequency_from_displayinfo_framerate(framerate: &str) -> Result<f32, std::num::ParseFloatError> {
```

Comment:
There's just a single place from where `get_frequency_from_displayinfo_framerate()` is called. Unless you have a clear intention to reuse this function in a code you're about to push in a near future, I'd rather move this couple of lines inside `get_rdk_resolution()`.

No need to pollute `interface.rs` with "functions" that aren't actually used as such.

Reply:
Thank you for the fix logic; pulled same into this to avoid merge conflict.